scout/constants/gene_tags.py

```diff
@@ -8,6 +8,8 @@
     ("XL", "XL - X Linked"),
     ("XD", "XD - X Linked Dominant"),
     ("XR", "XR - X Linked Recessive"),
+    ("Y", "Y-linked"),
```

Comment:
Checking what type of [models we are parsing from OMIM](https://github.com/Clinical-Genomics/scout/blob/fec5cd94f5cb800b0bc213072d90c78b9ab080af/scout/constants/disease_parsing.py#L17) I've found 2 more!